os_migrate/roles/conversion_host/tasks/main.yml

```diff
@@ -115,3 +115,8 @@
     ca_cert: "{{ os_migrate_conversion_ca_cert|default(omit) }}"
     client_cert: "{{ os_migrate_conversion_client_cert|default(omit) }}"
     client_key: "{{ os_migrate_conversion_client_key|default(omit) }}"
+
+- name: install os_migrate conversion host content
+  include_role:
+    name: conversion_host_content
+  when: os_migrate_conversion_host_content_install
```

Comment:
This is not the correct location to include the role - it would run it on the migrator host, not on conversion hosts. We should add this `include_role` at the end of `deploy_conversion_hosts.yml` playbook, in a similar manner like this:

https://github.com/os-migrate/os-migrate/blob/083d5796b5514aba335a08786cdc664644c813ea/os_migrate/playbooks/deploy_conversion_hosts.yml#L49-L54

This will give us the ability to specify which hosts it should run on.

Reply:
Fixed